src/ui/components/Comments/TranscriptComments/CommentEditor/ExistingCommentEditor.tsx

```diff
@@ -58,21 +52,10 @@ function ExistingCommentEditor({
         editable={isEditing}
         comment={comment}
         handleSubmit={inputValue => {
-          commentsLocalStorage.clear();
           setIsEditing(false);
           onSubmit(data, inputValue);
         }}
-        onCreate={({ editor }) => {
-          const storedComment = commentsLocalStorage.get();
-          if (storedComment) {
-            editor.commands.setContent(JSON.parse(storedComment));
-          }
-        }}
-        onUpdate={debounce(({ editor }) => {
-          commentsLocalStorage.set(JSON.stringify(editor.getJSON()));
-        }, PERSIST_COMM_DEBOUNCE_DELAY)}
```

Comment:
Just FYI this is definitely going to merge conflict with me because I'm using this place to update Redux, just a note for whichever of us merges second.